.github/workflows/build.yml

```diff
@@ -21,23 +21,14 @@ jobs:
     - run: npm ci
 
     # Build
-    - run: npm pack
+    - run: npm run pack
```

Comment:
This comment is for a top-level comment to discuss concerns / questions about the monorepo approach. Threads better than top-level comments in PR @TrevorKarjanis @jattasNI @mure @LukasKall